backend/examples/user_custom_properties/organization/locale/fr.json

```diff
@@ -0,0 +1,20 @@
+{
+  "User Info": "Information sur l'utilisateur",
+  "info about a user in this organization": "Information sur un utilisateur de cette organisation",
+  "Address": "Adresse",
+  "address of the user": "Adresse de l'utilisateur",
+  "First name": "Prénom",
+  "Last name": "Nom",
+  "Phone number": "Numéro de téléphone",
+  "Birthday date": "Date de naissance",
+  "Biography": "Biographie",
+  "Organization": "Organisation",
+  "name": "Nom",
+  "roles in the organization": "Role dans l'organisation",
+  "Developer": "Développeur",
+  "System Administrator": "Administrateur système",
+  "Manager": "Manager",
+  "Administrator": "Administrateur",
+  "Designer": "Designeur",
+  "Give use some informations about you.": "Parlez nous de vous."
```

Comment:
Why some english key use lowercase and some uppercase ?

Reply:
Just an issue. I splitted the test and the "more real" example in order to allow someone else to fix this example later.

Reply:
Information is invariable in English

Reply:
updated